### PR TITLE
Release AWS v12.3.0

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward giantswarm/giantswarm#11748

### CODE REVIEW

File: aws/v12.3.0/README.md
Comment (tomahawk28) [inline aws/v12.3.0/README.md:28]: Updated to the latest version for app-operator, chart-operator.

**Review by rossf7 — APPROVED** [review]
LGTM

**Comment by tomahawk28** [comment]
Still await for one of @giantswarm/sig-product approval and we will go with this.

File: README.md
Comment (alex-dabija) [inline README.md:13]: This should be under v12->v12.3, not under v12->v12.2 .

Comment (tomahawk28) [reply README.md:13]: Good catch, I fixed it now.

File: aws/v12.3.0/release.diff
Comment (alex-dabija) [inline aws/v12.3.0/release.diff:9]: The formatting of this file is a bit wonky. It looks like it contains a mix of spaces and tabs. I checked the previous PRs which all use spaces and are nicely displayed. It's not a big issue, but it would improve the readability of the file.

I use the following command (from devctl) to manually generate the release.diff file:
`diff aws/v12.1.4/release.yaml aws/v12.2.0/release.yaml -y -t > aws/v12.2.0/release.diff`

-t expands tabs to spaces

Comment (tomahawk28) [reply aws/v12.3.0/release.diff:9]: I will try your command above, I think I was doing without `-t` option.

File: aws/v12.3.0/README.md
Comment (cokiengchiara) [inline aws/v12.3.0/README.md:23]:
```suggestion
Please persist this note and the above, until all customers are on AWS v12.3.x and above.
```

It should persist until they are on **this** release, which is 12.3.x right? :)

**Review by alex-dabija — APPROVED** [review]

File: aws/v12.3.0/README.md
Comment (alex-dabija) [inline aws/v12.3.0/README.md:23]: There is another note which needs to be included from v12.1.0 because of the cert-manager changes. See: https://github.com/giantswarm/releases/tree/master/aws/v12.1.0 .

File: aws/v12.3.0/README.md
Comment (cokiengchiara) [reply aws/v12.3.0/README.md:28]: Jihyuk, which chart operator contains the fix which computes hash values correctly? 
Can you confirm v12.2.x has that?

File: aws/v12.3.0/README.md
Comment (cokiengchiara) [inline aws/v12.3.0/README.md:17]: When we did the preview release, we discovered the known issue about `deployed` vs `DEPLOYED` causing problems w MS Teams notification. Should we add that as a section here https://intranet.giantswarm.io/docs/dev-and-releng/helm/helm3-tenant-cluster-upgrade/?

File: aws/v12.3.0/README.md
Comment (tomahawk28) [reply aws/v12.3.0/README.md:23]: Yes, now we have two notes to SEs, they have different due date so I think it's better to separate them.

File: aws/v12.3.0/README.md
Comment (tomahawk28) [reply aws/v12.3.0/README.md:17]: Good idea, I will add that information into our doc.

File: aws/v12.3.0/README.md
Comment (tomahawk28) [reply aws/v12.3.0/README.md]: Thanks, I added it, too.

File: aws/v12.3.0/README.md
Comment (tomahawk28) [reply aws/v12.3.0/README.md:31]: @cokiengchiara It's 0.13.2 version and it is listed in AWS v12.2.0, Thanks for checking. 
https://github.com/giantswarm/releases/blob/ca506dc57583eb0c680689182b23e48d50648c58/aws/v12.2.0/release.yaml#L15-L16

**Comment by tomahawk28** [comment]
@cokiengchiara PTAL

**Review by cokiengchiara — APPROVED** [review]
LGTM when this is merged :D https://github.com/giantswarm/giantswarm/pull/13561

**Comment by cokiengchiara** [comment]
@tomahawk28 sorry, can you wait until at least 1pm? Want to check w sig-product. There shouldn't be any complications.

**Comment by cokiengchiara** [comment]
> @tomahawk28 sorry, can you wait until at least 1pm? Want to check w sig-product. There shouldn't be any complications.

No comments in the channel. LGTM.

**Pull request merged**
